docs/source/using/profiles.adoc

```diff
@@ -0,0 +1,37 @@
+= Minishift Profiles
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+
+[[create-multi-instances-of-minishift]]
+== Create multiple instances of Minishift
+
+With the help of profile feature you can create multiple instances of Minishift with xref:../command-ref/minishift_start.adoc#[`minishift start --profile`] command.
+
+[[manage-multi-instances-of-minishift]]
+== Manage multiple instances of Minishift
+
+When you have multiple profiles you can run Minishift commands  against a specific profile using `--profile` flag.
+
+If you do not want to pass `--profile` flag everytimei, you can activate a profile persistently with `minishift profile set` command.
+`minishift profile set`  is also used to switch between profiles.
+
+The default profile is `minishift`. `minishift start` and `minishift start --profile minishift` will create the default minishift profile.
```

Comment:
This documentation seems to be too concise. Definitely, it can be broken in to subsection 
- Starting with new profile
- List available profiles
- Switching to another profile

A specific note or section on default profile behavior. Like one can't delete default profile etc.